src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [h.name for h in humans if h.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [h.name for h in humans if h.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
-# whose name starts with any letter between 'C' and 'G' inclusive.
+# whose name starts with any letter between 'C' and 'G' inclusive. 103 - 107
 print("Starts between C and G, inclusive:")
-c = []
+c = [h.name for h in humans if h.name[0] in ("C", "D", "E", "F", "G")]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [h.age + 10 for h in humans ]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [f"{h.name}-{h.age}" for h in humans ]
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-f = []
+f = [(h.name, h.age) for h in humans if h.age in range(28, 33)]
```

Comment:
Range creates a list like this: `[28, 29, 30, 31, 32]` it's non-inclusive
`range(0, 10) => [0,1,2,3,4,5,6,7,8,9]`